gits/terminal.py

```diff
@@ -342,10 +342,13 @@ def _cap_dim(self):
         self._cap_set_color(2)
 
     def _cap_dl(self, n):
-        """Deletes ``n`` number of lines. """
+        """Deletes ``n`` number of lines in their entirety, with the lines
+        below moving up to fill the gap that is left.
+        """
```

Comment:
I get it=) and mentioned here both `dl` and `dl1`. The difference between them is very small. That's why `dl1` can use only `dl`.